restapi.go

```diff
@@ -2125,21 +2125,11 @@ func (s *Session) WebhookDeleteWithToken(webhookID, token string) (st *Webhook,
 // webhookID: The ID of a webhook.
 // token    : The auth token for the webhook
 // wait     : Waits for server confirmation of message send and ensures that the return struct is populated (it is nil otherwise)
-func (s *Session) WebhookExecute(webhookID, token string, wait bool, threadID string, data *WebhookParams) (st *Message, err error) {
+func (s *Session) WebhookExecute(webhookID, token string, wait bool, data *WebhookParams) (st *Message, err error) {
 	uri := EndpointWebhookToken(webhookID, token)
 
-	v := url.Values{}
-
 	if wait {
-		v.Set("wait", "true")
-	}
-
-	if threadID != "" {
-		v.Set("thread_id", threadID)
-	}
```

Comment:
We still need to add the thread ID in, as it's not part of the body; it's a query parameter.